src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -226,7 +227,10 @@
                 [buttonLoading]="!!exercise.loading"
                 [smallButton]="smallButtons"
                 [hideLabelMobile]="false"
-                *ngIf="gradedParticipation?.initializationState === InitializationState.INACTIVE && isResumeExerciseAvailable()"
+                *ngIf="
+                    (gradedParticipation?.initializationState === InitializationState.INACTIVE || gradedParticipation?.initializationState === InitializationState.FINISHED) &&
+                    isResumeExerciseAvailable()
```

Comment:
Should we move the initializationState-check also into `isResumeExerciseAvailable()`?

Reply:
I left this out, so the method is called as less as possible. And since the participation is often present and active while the student is working, the method has to be called almost never